scripts/service-startup-check.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+
+# zenoss-inspector-info
+# zenoss-inspector-tags os verify
+# zenoss-inspector-deps serviced-service-list-v.sh
+
+def main():
+    with open("serviced-service-list-v.sh.stdout", 'r') as f:
+        lines = f.readlines()
+    for line in lines:
+        if "Startup" and "/opt/zenoss/ZenPacks" in line:
+            print 'There is a problem with the Startup line for the following:'
```

Comment:
Good catch, that tag totally doesn't need to be there, other than that are
you signing off?


On Tue, Jul 19, 2016 at 6:14 AM, Glenn Jones <notifications@github.com>
wrote:

> In scripts/service-startup-check.py
> <https://github.com/zenoss/inspector/pull/45#discussion_r71318916>:
>
> > @@ -0,0 +1,18 @@
> > +#!/usr/bin/env python
> > +
> > +# zenoss-inspector-info
> > +# zenoss-inspector-tags os verify
> > +# zenoss-inspector-deps serviced-service-list-v.sh
> > +
> > +def main():
> > +    with open("serviced-service-list-v.sh.stdout", 'r') as f:
> > +        lines = f.readlines()
> > +    for line in lines:
> > +        if "Startup" and "/opt/zenoss/ZenPacks" in line:
> > +            print 'There is a problem with the Startup line for the following:'
>
> I think we should try to identify the parent service.
> The output from serviced-service-list is a JSON array, so instead of
> parsing the raw the ASCII, you could marshall the file contents into an
> array of JSON objects, then it would trivial to identify the services by
> name and/or service ID.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/zenoss/inspector/pull/45/files/2949ae684d5d5dda84c8b41c11a27cf69e5f51f6#r71318916>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AD4nWYLc0muQB1idleZ9YyUkUOyG3upFks5qXLGdgaJpZM4JPcnc>
> .
>



-- 
Christopher Loss
Zenoss Client Services Engineer